src/status_im/contexts/shell/jump_to/events.cljs

```diff
@@ -83,30 +83,35 @@
 (rf/defn navigate-to-jump-to
   {:events [:shell/navigate-to-jump-to]}
   [{:keys [db]}]
-  (let [open-floating-screens (shell.utils/open-floating-screens)]
-    {:db
-     (cond-> db
-
-       (get open-floating-screens shell.constants/chat-screen)
-       (assoc-in [:shell/floating-screens shell.constants/chat-screen :animation]
-        shell.constants/close-screen-with-shell-animation)
-
-       (and (get open-floating-screens shell.constants/chat-screen)
-            (get open-floating-screens shell.constants/community-screen))
-       (assoc-in [:shell/floating-screens shell.constants/community-screen :animation]
-        shell.constants/close-screen-without-animation)
-
-       (and (not (get open-floating-screens shell.constants/chat-screen))
-            (get open-floating-screens shell.constants/community-screen))
-       (assoc-in [:shell/floating-screens shell.constants/community-screen :animation]
-        shell.constants/close-screen-with-shell-animation)
-
-       (get open-floating-screens shell.constants/discover-communities-screen)
-       (assoc-in [:shell/floating-screens shell.constants/discover-communities-screen :animation]
-        shell.constants/close-screen-without-animation))
-
-     :dispatch [:set-view-id :shell]
-     :effects.shell/navigate-to-jump-to nil}))
+  (let [current-view-id       (:view-id db)
+        open-floating-screens (shell.utils/open-floating-screens)]
+    (assoc
```

Comment:
this whole block feels like it could be structured a bit easier to read.
Anyway, at the very least I think we should put the `(get open-floating-screens ...)` in a let var as it becomes a bit easier to read imo

e.g
```

(let [chat-open? (get open-floating-screens shell.constants/chat-screen)
      community-open?  (get open-floating-screens shell.constants/community-screen)
      discover-open? (get open-floating-screens shell.constants/discover-communities-screen)]
(assoc
   (if (shell.utils/shell-navigation? current-view-id)
     {:db
      (cond-> db

        chat-open?
        (assoc-in [:shell/floating-screens shell.constants/chat-screen :animation]
                  shell.constants/close-screen-with-shell-animation)

        (and chat-open?
             community-open?)
        (assoc-in [:shell/floating-screens shell.constants/community-screen :animation]
                  shell.constants/close-screen-without-animation)

        (and (not chat-open?)
             community-open?)
        (assoc-in [:shell/floating-screens shell.constants/community-screen :animation]
                  shell.constants/close-screen-with-shell-animation)

        discover-open?
        (assoc-in [:shell/floating-screens shell.constants/discover-communities-screen :animation]
                  shell.constants/close-screen-without-animation))

      :dispatch [:set-view-id :shell]}
     {:dispatch [:pop-to-root :shell-stack]})
   :effects.shell/navigate-to-jump-to
   nil))
 ```

Reply:
done, thank you